aperturedb/ParallelQuery.py

```diff
@@ -140,6 +110,44 @@ def setSuccessStatus(cls, statuses: list[int]):
     def getSuccessStatus(cls):
         return cls.success_statuses
 
+    @classmethod
+    def map_response_to_handler(cls, handler, query, query_blobs,  response, response_blobs,
+                                commands_per_query, blobs_per_query):
+        # We could potentially always call this handler function
+        # and let the user deal with the error cases.
+        blobs_returned = 0
+        for i in range(math.ceil(len(query) / commands_per_query)):
+            start = i * commands_per_query
+            end = start + commands_per_query
+            blobs_start = i * blobs_per_query
+            blobs_end = blobs_start + blobs_per_query
+
+            b_count = 0
+            if issubclass(type(response), list):
+                for req, resp in zip(query[start:end], response[start:end]):
+                    for k in req:
+                        # Ref to https://docs.aperturedata.io/query_language/Reference/shared_command_parameters/blobs
+                        blobs_where_default_true = \
```

Comment:
This is not correct anymore. blobs=False is default behavior

Reply:
fixed.